src/Access/MultipleAccessStorage.cpp

```diff
@@ -27,6 +28,14 @@ MultipleAccessStorage::MultipleAccessStorage(const String & storage_name_)
 {
 }
 
+MultipleAccessStorage::~MultipleAccessStorage()
+{
+    const auto storages = getStoragesPtr();
```

Comment:
/// It's better to remove storages in the reverse order because they could depend on each others somehow.

Reply:
Added.